test/edu/ucsb/nceas/metacatnettest/README.md

```diff
@@ -0,0 +1,7 @@
+# Use the `test/edu/ucsb/nceas/metacatnettest` directory only for tests that require a running metacat instance
```

Comment:
Same comment as above on naming. Not needed for this PR, but consider it for Metacat 3.0 in a testing refactor.